manifests/plugin.pp

```diff
@@ -9,10 +9,14 @@
 #
 # @param manage_repo
 #   Set to false if the plugin source repository is managed externally.
+#
+# @param extra_gemfiles
+#   Additional gemfiles a plugin needs added
 define katello_devel::plugin (
   Optional[String] $settings_template = undef,
   Optional[String] $scm_revision = undef,
   Boolean $manage_repo = true,
+  Array[String] $extra_gemfiles = [],
```

Comment:
It has been a long time since I looked at puppet, here we are setting an empty array, then in https://github.com/theforeman/puppet-katello_devel/pull/307/files#diff-b55cd2d4fee4ad88a9e21316ff2015c5579e8ec9bef7bc0fe5253262905fa702R51 we are hard coding it, is that so Katello always has it and we can override it if needed?

Reply:
This is a `define` which is meant to be a re-usable chunk of code, so we assume this is empty by default and plugin definitions can pass in values if they need.